src/client/src/ui/spotTile/views/spotTileView.jsx

```diff
@@ -28,6 +37,16 @@ export default class SpotTileView extends ViewBase {
     let spotDateClass = classnames('spot-tile__delivery', {'hide': model.hasNotification});
     let notification = this._tryCreateNotification();
     let priceComponents = this._createPriceComponents();
+    let showChartIQIcon = model._executionService._openFin.available;
```

Comment:
this is pretty hacky, the model should do the proxying. And even they it shouldn't access 'private' variables of other objects